gossipsub.go

```diff
@@ -1240,7 +1239,6 @@ func (gs *GossipSubRouter) heartbeatTimer() {
 }
 
 func (gs *GossipSubRouter) heartbeat() {
-	defer log.EventBegin(gs.p.ctx, "heartbeat").Done()
```

Comment:
???

Reply:
Event tracing through the logger is deprecated.